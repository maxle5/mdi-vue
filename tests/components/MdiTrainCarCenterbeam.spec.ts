
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiTrainCarCenterbeam from "../../src/components/MdiTrainCarCenterbeam.vue";

test("MdiTrainCarCenterbeam snapshot", () => {
  const wrapper = mount(MdiTrainCarCenterbeam, {});
  expect(wrapper.html()).toMatchSnapshot();
});
