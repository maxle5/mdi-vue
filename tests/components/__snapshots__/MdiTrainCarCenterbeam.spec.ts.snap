// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiTrainCarCenterbeam snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M23 6V17H22C22 18.11 21.11 19 20 19S18 18.11 18 17H6C6 18.11 5.11 19 4 19S2 18.11 2 17H1V6H3V15H21V6H23Z" style="fill: currentColor;"></path>
</svg>"
`;
